contrib/resources/mapping/MAIN.TXT

```diff
@@ -0,0 +1,2428 @@
+0xf3 0x0178        #LATIN CAPITAL LETTER Y WITH DIAERESIS
+0xf7 0x017e        #LATIN SMALL LETTER Z WITH CARON
+0xf9 0x0161        #LATIN SMALL LETTER S WITH CARON
+EXTENDS CODEPAGE 850
+
+CODEPAGE 860 # Portuguese
+EXTENDS FILE mapping-unicode.org CP_860.TXT
+
+CODEPAGE 861 # Icelandic
+EXTENDS FILE mapping-unicode.org CP_861.TXT
+
+CODEPAGE 862 # Hebrew-2
+EXTENDS FILE mapping-unicode.org CP_862.TXT
+
+CODEPAGE 863 # Canadian French
+EXTENDS FILE mapping-unicode.org CP_863.TXT
+
+CODEPAGE 864 # Arabic
+# 0x9b TODO: unidentified character - FreeDOS specific
+# 0x9c TODO: unidentified character - FreeDOS specific
+# 0x9f TODO: unidentified character - FreeDOS specific
+# 0xa6 TODO: unidentified character - FreeDOS specific
+# 0xa7 TODO: unidentified character - FreeDOS specific
+EXTENDS FILE mapping-unicode.org CP_864.TXT
+
+CODEPAGE 865 # Nordic
+EXTENDS FILE mapping-unicode.org CP_865.TXT
+
+CODEPAGE 866 # Russian, Cyrillic
+# Note: this is NOT an alternative layout mentioned here:
+# https://en.wikipedia.org/wiki/Code_page_866
+EXTENDS FILE mapping-unicode.org CP_866.TXT
+
+CODEPAGE 867 # Czech and Slovak, Kamenicky encoding
+# reference: https://en.wikipedia.org/wiki/Kamenick%C3%BD_encoding
+0x80 0x010c        #LATIN CAPITAL LETTER C WITH CARON
+0x83 0x010f        #LATIN SMALL LETTER D WITH CARON
+0x85 0x010e        #LATIN CAPITAL LETTER D WITH CARON
+0x86 0x0164        #LATIN CAPITAL LETTER T WITH CARON
+0x87 0x010d        #LATIN SMALL LETTER C WITH CARON
+0x88 0x011b        #LATIN SMALL LETTER E WITH CARON
+0x89 0x011a        #LATIN CAPITAL LETTER E WITH CARON
+0x8a 0x0139        #LATIN CAPITAL LETTER L WITH ACUTE
+0x8b 0x00cd        #LATIN CAPITAL LETTER I WITH ACUTE
+0x8c 0x013e        #LATIN SMALL LETTER L WITH CARON
+0x8d 0x013a        #LATIN SMALL LETTER L WITH ACUTE
+0x8f 0x00c1        #LATIN CAPITAL LETTER A WITH ACUTE
+0x91 0x017e        #LATIN SMALL LETTER Z WITH CARON
+0x92 0x017d        #LATIN CAPITAL LETTER Z WITH CARON
+0x95 0x00d3        #LATIN CAPITAL LETTER O WITH ACUTE
+0x96 0x016f        #LATIN SMALL LETTER U WITH RING ABOVE
+0x97 0x00da        #LATIN CAPITAL LETTER U WITH ACUTE
+0x98 0x00fd        #LATIN SMALL LETTER Y WITH ACUTE
+0x9b 0x0160        #LATIN CAPITAL LETTER S WITH CARON
+0x9c 0x013d        #LATIN CAPITAL LETTER L WITH CARON
+0x9d 0x00dd        #LATIN CAPITAL LETTER Y WITH ACUTE
+0x9e 0x0158        #LATIN CAPITAL LETTER R WITH CARON
+0x9f 0x0165        #LATIN SMALL LETTER T WITH CARON
+0xa4 0x0148        #LATIN SMALL LETTER N WITH CARON
+0xa5 0x0147        #LATIN CAPITAL LETTER N WITH CARON
+0xa6 0x016e        #LATIN CAPITAL LETTER U WITH RING ABOVE
+0xa7 0x00d4        #LATIN CAPITAL LETTER O WITH CIRCUMFLEX
+0xa8 0x0161        #LATIN SMALL LETTER S WITH CARON
+0xa9 0x0159        #LATIN SMALL LETTER R WITH CARON
+0xaa 0x0155        #LATIN SMALL LETTER R WITH ACUTE
+0xab 0x0154        #LATIN CAPITAL LETTER R WITH ACUTE
+0xad 0x00a7        #SECTION SIGN
+EXTENDS CODEPAGE 437
+
```

Comment:
We don't have CPX file containing screen font for it, so it would be next to useless. Plus, the code page information file (http://ftpmirror.your.org/pub/misc/ftp.software.ibm.com/software/globalization/gcoc/attachments/CP00868.txt) has a different format, so I would need to either extend the parser, or convert it manually.

For now, not worth the effort. I have added 874 only because it was a part of the set, and the file format was the same, so the effort was minimal.